internal/command/auth/webauth/webauth.go

```diff
@@ -76,6 +82,82 @@ func RunWebLogin(ctx context.Context, signup bool) (string, error) {
 	return token, nil
 }
 
+func StartCLISessionWebAuth(machineName string, signup bool) (fly.CLISession, error) {
+
+	return StartCLISession(machineName, map[string]interface{}{
+		"signup": signup,
+		"target": "auth",
+	})
+}
+
+// StartCLISession starts a session with the platform via web
+func StartCLISession(sessionName string, args map[string]interface{}) (fly.CLISession, error) {
+	var result fly.CLISession
+
+	if args == nil {
+		args = make(map[string]interface{})
+	}
+	args["name"] = sessionName
+
+	postData, _ := json.Marshal(args)
+
+	url := fmt.Sprintf("%s/api/v1/cli_sessions", "http://localhost:4000")
```

Comment:
I've only needed to copy all this code from `fly-go` because I needed to change this second argument